### ISSUE TITLE
False alarm of INPC005

### ISSUE THREAD

**Issue by nestquik** [DotNetAnalyzers/PropertyChangedAnalyzers#27]
```csharp
public class ViewModel : PropertyChangedBase
{
    private int _anotherCoolProp;

    public int AnotherCoolProp
    {
        get => _anotherCoolProp;
        set
        {
            if (!Set(ref _anotherCoolProp, value))
            {
                return;
            }

            NotifyOfPropertyChange();
        }
    }
}
```

**Comment by JohanLarsson** [comment]
Do you mean:
```cs
namespace RoslynSandbox
{
    public class ViewModel : Caliburn.Micro.PropertyChangedBase
    {
        private string name;

        public string Greeting => $""Hello {this.Name}"";

        public string Name
        {
            get { return this.name; }
            set
            {
                if (!Set(ref _anotherCoolProp, value))
                {
                    return;
                }

                this.NotifyOfPropertyChange(nameof(this.Greeting));
            }
        }
    }
}
```
